sal_interfaces/Electrometer/Electrometer_Events.xml

```diff
@@ -84,28 +86,29 @@
 <SALEvent>
     <Subsystem>Electrometer</Subsystem>
     <Version>3.8</Version>
-    <Author></Author>
+    <Author>aanania</Author>
     <EFDB_Topic>Electrometer_logevent_intensity</EFDB_Topic>
     <Alias>intensity</Alias>
+	<Description>Intensity of light measured with the photo-diode in the unit previously configured</Description>
```

Comment:
Oh... if this works for all units then it really shouldn't be called intensity.
maybe CurrentValue?
I realize this ticket is just about descriptions, but can we log this as something to change in the future?